packages/components/fields/async-creatable-select-field/src/async-creatable-select-field.tsx

```diff
@@ -387,6 +388,7 @@ export default class AsyncCreatableSelectField extends Component<
     const hasError =
       AsyncCreatableSelectInput.isTouched(this.props.touched) &&
       hasErrors(this.props.errors);
+    const errorFieldId = randomString();
```

Comment:
I would suggest to use the `createSequentialId` instead of using a new function.